src/utilities/session.ts

```diff
@@ -117,26 +136,50 @@ export class Session {
    * @returns - A promise containing a boolean, returning true if the user has access
    */
   async validateOpenShiftAccess(): Promise<boolean> {
-    const k8s = new KubernetesContext();
-    if (k8s?.coreV1Api) {
-      return k8s.coreV1Api
-        .readNamespace(k8s.namespace)
-        .then(() => {
+    const k8s = new KubernetesObj();
```

Comment:
I modified this function to use the same in the `KubernetesObj` as you mentioned to keep consistency and more maintainable, thank you!!